src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,26 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a `clone` method might not be necessary unless explicitly required for the project. If cloning of entities is a requirement, consider whether using a copy constructor or a static factory method to create copies of Movie objects would be more appropriate. Cloning complex objects with relationships can lead to unexpected behavior, especially when involving managed entities with ORM frameworks like Hibernate.